src/nway/onprem_utils/README.md

```diff
@@ -0,0 +1,33 @@
+### on-prem usage
+
+* install as you normally would, but include the optional `ONPREM`:
+```
+pip install .[ONPREM]
+```
+or
+```
+pip install -e .[ONPREM]
```

Comment:
Hmm... I'm not a fan of providing end users the option of installing in editable mode. It's a really useful dev feature, but can really complicate troubleshooting with end users.

Maybe just remove?

Reply:
done